### ISSUE TITLE
Links on page open as xref

### ISSUE THREAD

**Issue by abhipsaMisra** [MicrosoftDocs/feedback#1023]
**Describe the bug**
Unable to navigate to the links present in the page; the links are presented as xref

**To Reproduce**
1. Go to 'https://docs.microsoft.com/en-us/rest/api/iot-dps/'
2. Click on 'Device Enrollment' under 'REST Operation Groups'.
3. See error: A windows prompt saying "You'll need a new app to open this xref"

**Expected behavior**
Clicking on any of the operation group links should open a new page showing details of the selected operation group. Instead, we get a windows prompt saying "You'll need a new app to open this xref".

**Screenshots**

**Desktop (please complete the following information):**
 - OS: [e.g. iOS] Windows 10
 - Browser [e.g. chrome, safari] Microsoft Edge
 - Version [e.g. 22] 44.17763.1.0

**Smartphone (please complete the following information):**

**Additional context**

**Comment by mairaw** [comment]
@sethmanheim you seem to be listed as the owner of that page

**Comment by sethmanheim** [comment]
@mairaw No, definitely not. I haven't worked on IoT in a while -- you might check with Phil Meadows.

**Comment by mairaw** [comment]
@philmea do you know?

**Comment by philmea** [comment]
Thanks! I will verify the issue and escalate for a resolution as appropriate.

**Comment by mairaw** [comment]
Can we also update the metadata to reflect the right owner @philmea?
